books/free-programming-books-subjects.md

```diff
@@ -260,6 +260,7 @@ Books that cover a specific programming language can be found in the  [BY PROGRA
 * [Programming with gtkmm 4](https://developer.gnome.org/gtkmm-tutorial/stable/)
 * [Search User Interfaces](http://searchuserinterfaces.com/book/) - Marti A. Hearst
 * [The GLib/GTK+ Development Platform](https://people.gnome.org/~swilmet/glib-gtk-dev-platform.pdf) - Sébastien Wilmet (PDF)
+* [Web Design Primer](https://pressbooks.library.ryerson.ca/webdesign/) - Richard Adams, Ahmed Sagarwala
```

Comment:
- Providing platform... `(Ryerson University Pressbooks)` to fully satisfy with CC
- And available formats too

```markdown
* [Web Design Primer](https://pressbooks.library.ryerson.ca/webdesign/) - Richard Adams, Ahmed Sagarwala (HTML, PDF, EPUB, Kindle) (Ryerson University Pressbooks)
```